.gitignore

```diff
@@ -3,4 +3,7 @@ tmp/
 .pyc
```

Comment:
So this was a mistake on my part. Could you please change this line to be `*.pyc` and remove lines 7-9 that you added?